ofbizDemo/webapp/ofbizDemo/ProductionDemo/FindProductionRun.ftl

```diff
@@ -0,0 +1,69 @@
+<div>
+    <h1>Find Production Run</h1>
+    <a href="<@ofbizUrl>CreateProduction</@ofbizUrl>" target="_blank"><button>Create a  Production Run</button></a>
+    <br>
+    <br>
+</div>
+<div class="screenlet">
+    <div class="screenlet-title-bar">
+        <ul>
+            <li class="h3">${uiLabelMap.SearchOption}</li>
+        </ul>
+        <br class="clear"/>
+    </div>
+    <div class="screenlet-body">
+        <div>
+            <form name="FindProductionRun" action="<@ofbizUrl>FindProductionRun</@ofbizUrl>" method="post">
+                <table width="100%">
+                    <tr>
+                        <td width="15%" align="right"><span class="label">${uiLabelMap.ProductionRunId}</span></td>
+                        <td width="2%"></td>
+                        <td width="83%"><input type="text" name="workEffortId" value="${parameters.workEffortId!}"/></td>
+                    </tr>
+                    <tr>
+                        <td width="15%" align="right"><span class="label">${uiLabelMap.Status}</span></td>
+                        <td width="2%"></td>
+                        <td width="83%">
+                            <#list statusItems as statusItem >
+                            <input type="checkbox" name="currentStatusId" value="${statusItem.statusId!}"/>${statusItem.description!}
```

Comment:
can you please add checked attribute if error occurs and user redirected to this page.